<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAP-04-1581" name="any test name1" tests="4" failures="1" timestamp="25 Aug 2016 13:39:50 GMT" time="0.05" errors="0">
  <testcase name="test1" time="0.007" classname="com.deena.TestNGLearn.YahooTest"/>
  <testcase name="test2" time="0.002" classname="com.deena.TestNGLearn.YahooTest"/>
  <testcase name="test3" time="0.002" classname="com.deena.TestNGLearn.YahooTest"/>
  <testcase name="test4" time="0.006" classname="com.deena.TestNGLearn.YahooTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [test3] but found [test4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [test3] but found [test4]
	at com.deena.TestNGLearn.YahooTest.test4(YahooTest.java:47)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
</testsuite> <!-- any test name1 -->
